### PR TITLE
[DOC Canary] Comments added for ember testing helper scripts

### PR DESCRIPTION
**Pull request by purvitiwari**
Documentation added according to #13862

### CODE REVIEW

File: packages/ember-testing/lib/helpers/find_with_assert.js [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+  var $el = findWithAssert('.doesnt-exist'); // throws error
+  ```
+  @method findWithAssert
+  @param {String} selector jQuery selector string for finding an element within
+  the DOM
+  @return {Object} jQuery object representing the results of the query
+  @throws {Error} throws error if jQuery object returned has a length of 0
+  @public
+*/
+
```
Comment (rwjblue) [inline packages/ember-testing/lib/helpers/find_with_assert.js]: Can you remove this blank line?

File: packages/ember-testing/lib/helpers/key_event.js [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+  @method keyEvent
+  @param {String} selector jQuery selector for finding element on the DOM
+  @param {String} type the type of key event, e.g. `keypress`, `keydown`, `keyup`
+  @param {Number} keyCode the keyCode of the simulated key event
+  @return {RSVP.Promise}
+  @since 1.5.0
+  @public
+*/
+
+
```
Comment (rwjblue) [inline packages/ember-testing/lib/helpers/key_event.js]: Can you remove these two blank lines?

**Comment by rwjblue** [comment]
Can you:

* add the following snippet to the top of each helper file:

```js
/**
@module ember
@submodule ember-testing
*/
```

* Remove the existing documentation in https://github.com/emberjs/ember.js/blob/master/packages/ember-testing/lib/helpers.js that this is replacing.

Thank you!

**Comment by rwjblue** [comment]
Can you squash commits and rebase against current master?

**Comment by mixonic** [comment]
@purvitiwari thank you!

**Pull request merged**